MAVProxy/tools/MAVExplorer.py

```diff
@@ -129,7 +129,7 @@ def __init__(self):
             "graph"     : ['(VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE)'],
             "dump"      : ['(MESSAGETYPE)', '--verbose (MESSAGETYPE)'],
             "map"       : ['(VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE) (VARIABLE)'],
-            "param"     : ['download', 'check', 'help (PARAMETER)'],
+            "param"     : ['download', 'check', 'help (PARAMETER)', 'save', 'savechanged', 'diff', 'show', 'check'],
```

Comment:
This looks better, but shouldn't some of these have ` (PARAMETER)` in them?

Reply:
its complex, some take wildcard or filenames